src/common/alertmanager/critical-alerts.service.ts

```diff
@@ -31,18 +31,9 @@ export class CriticalAlertsService {
     this.baseUrl = this.config.get('CRITICAL_ALERTS_ALERTMANAGER_URL') ?? '';
   }
 
-  private get alerts() {
-    return [
-      new CriticalNegativeDelta(this.config, this.storage),
-      new CriticalMissedProposes(this.config, this.storage),
-      new CriticalMissedAttestations(this.config, this.storage),
-      new CriticalSlashing(this.config, this.storage),
-    ];
-  }
-
-  public async sendCriticalAlerts(bySlot: bigint) {
+  public async send(epoch: bigint) {
     if (this.prometheus.getSlotTimeDiffWithNow() > 3600000) {
```

Comment:
I think the `getSlotTimeDiffWithNow` method should moved away from **Prometheus** service since it is a responsibility of **Inspector** service.

Reply:
I didn't quite understand your point. May be kept as is, I think.